src/pages/Actions/components/AddFunds.tsx

```diff
@@ -142,10 +142,14 @@ const AddFunds = ({ validator }: AddFundsProps) => {
   };
 
   const handleValueChange = (value: number) => {
-    const newValue = Math.max(
-      Math.min(value, etherMaxUseful),
-      MIN_DEPOSIT_ETHER
-    );
+    let newValue = Math.max(Math.min(value, etherMaxUseful), MIN_DEPOSIT_ETHER);
+
+    // Do not allow more than 1 gwei precision as required by the deposit contract
+    newValue = new BigNumber(value)
+      .multipliedBy(ETHER_TO_GWEI)
+      .integerValue(BigNumber.ROUND_DOWN)
+      .dividedBy(ETHER_TO_GWEI)
+      .toNumber();
```

Comment:
We're immediately mutating the `newValue` variable, and unconditionally, so the initial `let` assignment is not needed, and we can use `const` instead. The multiply => integer rounding => dividing can be equivilently done with `.decimalPlaces(9, BigNumber.ROUND_DOWN)`, so I would suggest this:

```suggestion
    // Round to gwei precision as required by the deposit contract
    const newValue = new BigNumber(value)
      .decimalPlaces(9, BigNumber.ROUND_DOWN)
      .toNumber();
```

Reply:
@wackerow thanks for the input, I changed it